pymc3/tests/test_distribution_defaults.py

```diff
@@ -1,11 +1,11 @@
 from __future__ import division
```

Comment:
I'd want to check on that carefully -- so long as we support python2, this makes sure `2 / 3 != 0`.  I've never counted on this myself (choosing to explicitly cast), but I think that would be for a different PR.